stagemonitor-requestmonitor/src/main/java/org/stagemonitor/requestmonitor/MonitoredMethodRequest.java

```diff
@@ -1,5 +1,6 @@
 package org.stagemonitor.requestmonitor;
 
+import com.google.common.util.concurrent.ListenableFuture;
```

Comment:
Guava is only available through the optional Elasticsearch dependency so we can't assume it's on the classpath. I'm quite reluctant to add it as a mandatory dependency for stagemonitor as guava is quite large and stagemonitor tries to have as few dependencies as possible. One more reason besides the size is that there is always the danger of conflicting versions especially because some application servers already have guava in the classpath. I'm sorry, but guava is a show stopper.